### ISSUE TITLE
[FEATURE] Python scripts for GuitarMLAmp?

### ISSUE THREAD

**Issue by GoodNoiseLLC** [Chowdhury-DSP/BYOD#149]
Hello,

Are python scripts available to train and create new JSON models for GuitarMLAmp?

Thanks!

**Comment by jatinchowdhury18** [comment]
Yes! Training scripts for the GuitarML neural networks are available in this repository. @GuitarML might be able to elaborate a little a little bit more on the latest and greatest way to train GuitarML models.
